src/Datadog.Trace.Tools.Runner/Utils.cs

```diff
@@ -119,6 +119,21 @@ internal class Utils
                 envVars["DD_TRACE_AGENT_URL"] = options.AgentUrl;
             }
 
+            if (!string.IsNullOrWhiteSpace(options.EnvironmentValues))
+            {
+                foreach (var keyValue in options.EnvironmentValues.Split(','))
```

Comment:
Do you want to auto-trim white space around entries here?

```suggestion
                foreach (var keyValue in options.EnvironmentValues.Split(',', StringSplitOptions.TrimEntries))
```